app/src/main/java/org/akvo/flow/service/ExceptionReportingService.java

```diff
@@ -153,6 +153,8 @@ public void submitStackTraces(String server) {
                     File f = new File(dir, list[i]);
                     String trace = FileUtil.readFileAsString(f);
 
+                    // We cannot use the standard FlowApi.getDeviceParams, fot this service uses
+                    // a different naming convention...
                     Map<String, String> params = new HashMap<String, String>();
```

Comment:
Apparently this servlet uses it's own convention, thus it is probably better off without any extra complexity. However, it might be a good idea to send the `androidId` param, even if its not read.